tests/asr_benchmark_test.py

```
@@ -0,0 +1,280 @@
+    measure_transcription_time,
+    print_gpu_info,
+    print_performance_summary,
+    save_benchmark_results,
+)
+
+
+class TestASRBenchmark:
+    """Benchmark tests for ASR"""
+
+    @pytest.mark.benchmark
+    def test_single_transcription_performance(self, asr_model_warmed, test_audio_files, benchmark_results_dir):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_single_transcription_performance`

Add return type annotation: `None`

(ANN201)